Cargo.toml

```diff
@@ -272,6 +272,14 @@ wasmi = { opt-level = 3 }
 x25519-dalek = { opt-level = 3 }
 yamux = { opt-level = 3 }
 zeroize = { opt-level = 3 }
+
 [profile.release]
 # Substrate runtime requires unwinding.
 panic = "unwind"
+
+# Sacrifice compile speed for execution speed by using optimization flags:
+
+# https://doc.rust-lang.org/rustc/linker-plugin-lto.html
+lto = "fat"
+# https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units
+codegen-units = 1
```

Comment:
any reason for not using custom profiles as was done in https://github.com/paritytech/polkadot/pull/4627?
the tests in CI don't need super slow compilation with LTO and codegen-units = 1

Reply:
Thanks for the comment! I was actually looking to discuss this.  

No there is no specific reason for this. I guess a custom profile would be less pain to use for developers.  
Waiting if someone else has ideas before I update it.

Reply:
Yeah, we really don't want to add this as default to the `Cargo.toml`.

Reply:
Moved it to a `production` profile.

Reply:
Great, rust `1.56.0` in the CI does not yet support custom profiles :grimacing:

Reply:
@bkchr If you want to have the honor of the final approval, please go ahead.